skyline-vscode/react-ui/src/Habitat.js

```diff
@@ -36,30 +36,30 @@ export default function Habitat({habitatData}) {
     //     {y: 'device 4', x: 20, color: 3}
     // ];
 
-    const sampleDataVGG = [
-        {y: "V100", x: 38.6031339393715, color: 0},
-        {y: "P100", x: 67.57987999101333, color: 1},
-        {y: "T4", x: 118.20070742178441, color: 2},
-        {y: "P4000", x: 145.73762440806306, color: 3},
-        {y: "RTX2070", x: 74.75586879514157, color: 4}
-    ];
+    // const sampleDataVGG = [
```

Comment:
Can you delete this instead of commenting it out?

Reply:
Sure, on it.